### PR TITLE
[14.0] [MIG] password_security: Migrated the module in v14.

### PR DESCRIPTION
**Pull request by Chandresh-OSI**

### CODE REVIEW

**Comment by dreispt** [comment]
@Chandresh-OSI remember that pre-commit/black and actual changes should be in two different commits.

**Comment by dreispt** [comment]
@Chandresh-OSI Is this ready? Note that Travis is red.

**Comment by Chandresh-OSI** [comment]
Want to work on test cases, Travis is failing due to some test cases define in the module.

**Comment by CasVissers-360ERP** [comment]
Double effort? #244 

EDIT: Think this PR should replace #244

**Review by CasVissers-360ERP — APPROVED** [review]
Functional review.

File: password_security/views/res_company_view.xml [hunk 1]
```diff
@@ -1,39 +0,0 @@
...
-                        <group>
-                            <field name="password_numeric" />
-                            <field name="password_special" />
-                        </group>
-                    </group>
-                </page>
-            </xpath>
-        </field>
-    </record>
-</odoo>
```
Comment (dreispt) [inline password_security/views/res_company_view.xml]: Why is this being removed?

**Comment by dreispt** [comment]
Thank you for the review @CasVissers-360ERP.
Both this and #244 have red builds.
I would see which PR get merge ready fort.

**Comment by dreispt** [comment]
@Chandresh-OSI This is still pending closure.

**Comment by dreispt** [comment]
Added a commit to fix the user error messages.
Before and after the fix:

image

image

**Comment by dreispt** [comment]
Precommit fixed

**Comment by Chandresh-SerpentCS** [comment]
@dreispt Mr. Travis is Happy now. :)

**Review by CasVissers-360ERP — APPROVED** [review]
Functional test

**Review by dreispt — APPROVED** [review]

**Pull request merged**